site/zh/blog/new-in-devtools-99/index.md

```diff
@@ -31,165 +30,211 @@ draft: true
+在重放用户流程记录的过程中，现在的 **Recorder** 面板将会一直处于等待状态，直到元素在视口范围内能够被看见或可以被点击；或者 **Recorder** 面板会尝试自动将元素滚动到视口范围，然后再执行重放该步的动作。在此之前，重放动作会立即失败。
 
 <!-- Here is an example of an off-screen menu positioned outside of the viewport and slide in when activated. The user flow is to toggle the menu, and click on the menu item. Previously, the replay would fail at the last step, because the menu item is still sliding in and not visible in the viewport yet. It’s fixed now. -->
+这里有一个例子：菜单被放置在视口范围之外，当这个菜单被激活时，会从视口外面滑动到视口里面。这个用户流程描述了从点击 toggle off-screen menu 按钮开始到点击菜单里面选项的全过程。在此之前，重放动作会在最后一步失败，这是因为此时的菜单项还处于滑入状态且在视口范围内不可见。现在这个问题已经被修复。
 
 {% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/Qf8f2x1u1y5FEMSmkB3A.mp4", class="screenshot", autoplay=true, controls=true, loop=true, muted=true %}
 
-Chromium issue: [1257499](https://crbug.com/1257499#c38)
+Chromium 议题: [1257499](https://crbug.com/1257499#c38)
 
 
 <!-- ## Better console styling, formatting and filtering {: #console } -->
+## 更好的控制台样式，格式化和过滤 {: #console }
 
 <!-- ### Properly style log messages with ANSI escape code {: #console-styling }  -->
+### 使用 ANSI 转义码来适当地修饰日志信息 {: #console-styling }
 
 <!-- You can now use the [ANSI escape sequences](https://en.wikipedia.org/wiki/ANSI_escape_code) to properly style console messages. Previously, DevTools console had very limited (and partly broken) support for ANSI escape sequences. -->
+您现在可以使用 [ANSI escape sequences](https://en.wikipedia.org/wiki/ANSI_escape_code) 来适当地修饰控制台信息。在此之前，DevTools 控制台对 ANSI escape sequences 的支持实属有限（且部分功能已经失效）。
 
 <!-- It is common for [Node.js](https://nodejs.org/) developers to colorize log messages via ANSI escape sequences, often with the help of some styling libraries like [chalk](https://www.npmjs.com/package/chalk), [colors](https://www.npmjs.com/package/colors), [ansi-colors](https://www.npmjs.com/package/ansi-colors), [kleur](https://www.npmjs.com/package/kleur), etc.  -->
+对 [Node.js](https://nodejs.org/) 开发者来说，使用 ANSI escape sequences 来美化日志信息的做法实属平常，通常是需要使用一些样式库，如 [chalk](https://www.npmjs.com/package/chalk)、[colors](https://www.npmjs.com/package/colors)、[ansi-colors](https://www.npmjs.com/package/ansi-colors)、[kleur](https://www.npmjs.com/package/kleur) 等。
 
 <!-- With these changes, you can now debug your Node.js applications seamlessly using DevTools, with proper colorized console messages. Open this [demo](https://stackblitz.com/edit/node-colors-test) to view it yourself! -->
+有了这些改动，您现在就可以无差别地使用 DevTools 来调试您的 Node.js 应用程序，并且能够适当地美化控制台信息。打开这个 [demo](https://stackblitz.com/edit/node-colors-test) 来亲自目睹效果！
 
 <!-- To learn more about formatting & styling console messages with DevTools, go to [format and style messages in the Console](/docs/devtools/console/format-style) documentation. -->
+如果您想了解更多关于 DevTools 格式化以及 DevTools 修饰控制台信息的内容，请查看 [格式化和修饰控制台信息](/docs/devtools/console/format-style) 文档。
```

Comment:
Maybe change "关于 DevTools 格式化以及 DevTools 修饰控制台信息的内容" to "关于 DevTools 格式化以及美化控制台信息的内容", since we already translated "styling" to "美化" previously and after this.

Reply:
Okay